docs/en/benchmark/mdtest.md

```diff
@@ -6,17 +6,17 @@ slug: /mdtest
 # Benchmark with mdtest
 
 :::tip
-JuiceFS v1.0+ has Trash enabled by default, the benchmark will create and delete temporary files in the file system, these files will eventually be dumped to the Trash folder `.trash`, it will take up storage space, to avoid this, you can disable the Trash before the benchmark `juicefs config META-URL --trash-days 0` and refer to [trash](../security/trash.md) for details.
+JuiceFS v1.0+ has Trash enabled by default. The benchmark creates and deletes temporary files in the file system, and eventually dumps these files into a directory named `.trash`. To avoid storage space being occupied by `.trash`, you can run command `juicefs config META-URL --trash-days 0` to perform benchmark with Trash disabled. See [trash](../security/trash.md) for details.
```

Comment:
```suggestion
Trash is enabled in JuiceFS v1.0+ by default. As a result, temporary files are created and deleted in the file system during the benchmark, and these files will be eventually dumped into a directory named `.trash`. To avoid storage space being occupied by `.trash`, you can run command `juicefs config META-URL --trash-days 0` to disable Trash before benchmark. See [trash](../security/trash.md) for details.
```
稍微改了一下语序尝试着把句子变得更通畅一些，你看看行不行哈

Reply:
没问题 👌